inference-engine/samples/face_detection_java_sample/README.md

```diff
@@ -0,0 +1,53 @@
+# Face Detection Java Sample
+
+This guide describes how to run the java Face Detection sample application.
+
+## How It Works
+
+Upon the start-up the sample application reads command line parameters and loads a network and an image to the Inference
+Engine device. When inference is done, the application creates an output image and outputs data.
+
+To download model ( .bin and .xml files must be downloaded) use:
+https://download.01.org/opencv/2019/open_model_zoo/R1/models_bin/face-detection-adas-0001/FP32/
+
+## Build
+To create java library for Inference Engine add `-DENABLE_JAVA=ON` flag in cmake command while building dldt:
+```bash
+cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_JAVA=ON ..
+```
+
+Create an environment variable with OpenCV installation path:
+```bash
+export OCV_PATH=/path/to/opencv/build
+```
+
+and the same for Inference Engine:
+```bash
+export IE_PATH=/path/to/dldt/build
+```
+
+To compile sample use:
+```bash
+cd dldt/inference-engine/samples/face_detection_java_sample
+javac -cp ".:${OCV_PATH}/bin/opencv-420.jar:${IE_PATH}/inference-engine/ie_bridges/java/inference_engine_java_api.jar" Main.java
+```
+
+## Running
+Add library path for openvino java library before running:
+```bash
+export LD_LIBRARY_PATH=/home/<path_to_folder>/dldt/bin/intel64/Release/lib:$LD_LIBRARY_PATH
+```
+
+and for opencv library:
+```bash
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/<path_to_folder>/opencv/build/lib
+```
+
+To run sample use:
+```bash
+java -cp ".:${OCV_PATH}/bin/opencv-420.jar:${IE_PATH}/inference-engine/ie_bridges/java/inference_engine_java_api.jar" Main <image_path> face-detection-adas-0001.xml
+```
+
+## Sample Output
+
+The application outputs an image ('Detection.jpg') with detected objects enclosed in rectangles. It outputs the confidence value and the coordinates of the rectangle to the standard output stream.
```

Comment:
There is no `imwrite` in current version of the sample. Please change description (sample will show the results in new window)